pyproject.toml

```diff
@@ -50,4 +54,8 @@ typeCheckingMode = "basic"
 
 [tool.ruff]
 # See https://beta.ruff.rs/docs/configuration/
-select = ["E", "F", "I001"]
\ No newline at end of file
+select = ["E", "F", "I001"]
+
+[tool.unimport]
+sources = ["datahub_sap_hana", "tests"]
+exclude = '__init__.py|.venv/'
```

Comment:
oops, i forgot a new-line here, can you add one (just FYI that is what this red symbol means, a file ending without a trailing new-line)